app/pods/publications/publication/translations/template.hbs

```diff
@@ -1,23 +1,82 @@
-<Auk::Navbar @border="bottom">
-  <Auk::Toolbar>
-    <Auk::Toolbar::Group @position="left">
-      <Auk::Toolbar::Item>
-        <Auk::Tabs>
-          <Auk::Tab
-            data-test-route-publications---translations-tab-documents
-            @route="publications.publication.translations.documents"
-          >
-            {{capitalize (t "documents")}}
-          </Auk::Tab>
-          <Auk::Tab
-            data-test-route-publications---translations-tab-requests
-            @route="publications.publication.translations.requests"
-          >
-            {{capitalize (t "requests")}}
-          </Auk::Tab>
-        </Auk::Tabs>
-      </Auk::Toolbar::Item>
-    </Auk::Toolbar::Group>
-  </Auk::Toolbar>
-</Auk::Navbar>
-{{outlet}}
\ No newline at end of file
+<div class="auk-u-m-4">
+  <Publications::Publication::Translations::TranslationsInfoPanel
+    @translationSubcase={{this.translationSubcase}}
+    @isFinal={{this.publicationFlow.status.isFinal}}
+  />
```

Comment:
Kan je deze route nog verder opsplitsen zodat het info panel op `publications.publication.translations` staan en de activiteiten-timeline op de index subroute `publications.publication.translations.index` komt te staan. Op die manier gaan de model-hooks cleaner worden. 
Die structuur is nu ook gebruikt voor het info-panel vs de referentiedocumenten op de 'Beslissingen'-tab. Zie https://github.com/kanselarij-vlaanderen/frontend-kaleidos/tree/development/app/pods/publications/publication/decisions